docker/mongo/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mongo:3.6.3
+
+EXPOSE 27017
+
+ADD run.sh /run.sh
```

Comment:
Codacy found an issue: [Use COPY instead of ADD for files and folders](https://app.codacy.com/manual/impulsonetwork/atena/pullRequest?prid=6004068)